packages/ignition/src/app/structured-data/creative-work.ts

```diff
@@ -0,0 +1,21 @@
+import { Organization, Thing } from ".";
```

Comment:
Do not use import like that, it creates redundant references. Next can handle that, so, it works, but it's not ok